packages/opentelemetry-exporter-collector/src/types.ts

```diff
@@ -30,7 +31,7 @@ export namespace opentelemetryProto {
       export interface ExportTraceServiceRequest {
         resourceSpans: opentelemetryProto.trace.v1.ResourceSpans[];
       }
-
+      // eslint-disable-next-line @typescript-eslint/no-empty-interface
```

Comment:
@obecny Do we really need this(`ExportTraceServiceResponse`) empty interface?

Reply:
remove it this is not needed